clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10999,8 +10999,8 @@ def err_sycl_restrict : Error<
   "|use a const static or global variable that is neither zero-initialized "
   "nor constant-initialized"
   "}0">;
-def warn_sycl_kernel_too_many_args : Warning<
-  "kernel argument count (%0) exceeds supported maximum of %1 on GPU">,
+def warn_sycl_kernel_too_big_args : Warning<
+  "kernel arguments size (%0 bytes) exceeds supported maximum of %1 on GPU">,
```

Comment:
Suggestion:
"size of kernel arguments (%0 bytes) exceeds supported maximum of %1 bytes on GPU"

Reply:
Done.